lib/ret_web/controllers/api/v1/presence_controller.ex

```diff
@@ -0,0 +1,10 @@
+defmodule RetWeb.Api.V1.PresenceController do
+  use RetWeb, :controller
+
+  # Get presence count
+  def show(conn, _) do
+    count = RetWeb.Presence.present_session_count()
```

Comment:
I am a bit hazy on the difference between this and `present_member_count()`. From a quick glance it seems this will over count a bit since it will count users in the admin panel https://github.com/mozilla/hubs/blob/c5513b1f2d5f4a465189de569acbd4ea2a8e6edf/admin/src/admin.js#L279 and scene pages https://github.com/mozilla/hubs/blob/c5513b1f2d5f4a465189de569acbd4ea2a8e6edf/src/scene.js#L95 and briefly on other pages as part of checking if a user is an admin https://github.com/mozilla/hubs/blob/c5513b1f2d5f4a465189de569acbd4ea2a8e6edf/src/react-components/auth/AuthContext.js#L14 ... Member count conceptually seems more correct though it seems that only includes users not in the lobby. We might need a new stat for this.

That said this number is probably close enough to be useful to start and is easy to change later, so up to you if you feel like changing it here. I'm sure requirements will evolve to needing more from this response at some point anyway, might make more sense to tackle there with more clarity.

Reply:
Verified, that it does count users in the admin panel.

Reply:
Tested brian's code. A scene page does not factor into the in room ccu count.